js/stateless.js/src/rpc.ts

```diff
@@ -153,39 +153,46 @@ async function getCompressedTokenAccountsByOwnerOrDelegate(
 }
 
 /** @internal */
-function buildCompressedAccountWithMaybeTokenData(account: any): {
+function buildCompressedAccountWithMaybeTokenData(
+    accountStructWithOptionalTokenData: any,
+): {
     account: CompressedAccountWithMerkleContext;
     maybeTokenData: TokenData | null;
 } {
-    const tokenData = account.optionTokenData;
+    const compressedAccountResult = accountStructWithOptionalTokenData.account;
+    const tokenDataResult =
+        accountStructWithOptionalTokenData.optionalTokenData;
+
     const compressedAccount: CompressedAccountWithMerkleContext =
         createCompressedAccountWithMerkleContext(
             createMerkleContext(
-                account.tree!,
+                compressedAccountResult.merkleTree,
                 mockNullifierQueue,
-                account.hash.toArray(undefined, 32),
-                account.leafIndex,
+                compressedAccountResult.hash.toArray(undefined, 32),
```

Comment:
I think we should be explicit here whether to convert to "be" or "le" instead of undefined.